utils/release_helper.py

```diff
@@ -0,0 +1,786 @@
+    next_release_date = get_next_release_date(latest_release.published_at)
+    date = get_date_for_message(next_release_date)
+    branch = get_next_stabilization_branch(repo)
+
+    future_version = bump_version(next_release_version, 'minor')
+    future_release_date = get_next_release_date(next_release_date)
+    future_date = get_date_for_message(future_release_date)
+    future_stabilization_date = get_next_stabilization_date(future_release_date)
+    future_date_stabilization = get_date_for_message(future_stabilization_date)
+
+    template = f'''
+        Subject: stabilization of v{next_release_version}
+
+        Hello all,
+
+        The release of Content version {next_release_version} is scheduled for {date}.
+        As part of the release process, a stabilization branch was created.
+        Issues and PRs that were not solved were moved to the {future_version} milestone.
+
+        Any bug fixes you would like to include in release {next_release_version} should be
+        proposed for the *{branch}* and *master* branches as a measure to avoid
+        potential conflicts between these branches.
+
+        The next version, {future_version}, is scheduled to be released on {future_date},
+        with the stabilization phase starting on {future_date_stabilization}.
+
+        Regards,'''
+    return template
+
+
+def get_release_end_message(repo) -> str:
+    latest_release = get_latest_release(repo)
+    latest_release_url = latest_release.html_url
+    highlights = get_release_highlights(latest_release)
+    last_commit = get_contributors_last_commit()
+    new_contributors = get_new_contributors(last_commit)
+    released_version = get_latest_version(repo)
+
+    for asset in latest_release.get_assets():
+        if asset.content_type == 'application/x-bzip2':
+            source_tarball = asset.browser_download_url
+        elif asset.content_type == 'application/zip':
+            prebuild_zip = asset.browser_download_url
+        else:
+            if '.tar.bz2.sha512' in asset.name:
+                source_tarball_hash = asset.browser_download_url
+            else:
+                prebuild_zip_hash = asset.browser_download_url
+
+    # It would be more readable if this multiline string would be indented. However, this makes
+    # the final text weird, after the "format" substitutions, when highlights and new_contributors
+    # have multilines too. So, the readability was compromised in favor of simplicity and user
+    # experience. The user only needs to copy and paste, without removing leading spaces.
+    template = f'''
+Subject: ComplianceAsCode/content v{released_version}
+
+Hello all,
+
+ComplianceAsCode/Content v{released_version} is out.
+
+Some of the highlights of this release are:
+{highlights}
+
+Welcome to the new contributors:
+{new_contributors}
+
+For full release notes, please have a look at:
+{latest_release_url}
+
+Pre-built content: {prebuild_zip}
+SHA-512 hash: {source_tarball_hash}
```

Comment:
I think that you mismatched the hashes here, see few lines below.

Reply:
Correct

Reply:
Done.